src/_includes/docs/learning-resources-index/side-filters.liquid

```diff
@@ -0,0 +1,39 @@
+<div id="resource-filter-group">
+    <div class="table-title">Filter by</div>
+    <div class="table-content">
+        <h4>Subject</h4>
+        <ul id="filters-tags" class="collapsed">
+            {% for filter in filters.tags -%}
+                <li class="hidden">
+                    <input type="checkbox"
+                           role="checkbox"
+                           name="filter-{{filter}}"
+                           data-category="tags"
+                           id="filter-{{filter}}">
+                    <label for="filter-{{filter}}">{{filter}}</label>
+                </li>
+            {% endfor -%}
+        </ul>
+        <button id="filters-tags-show-button">
+            <span class="label">More</span>
+            <span class="material-symbols" aria-hidden="true">expand_more</span>
+        </button>
+        <h4>Type</h4>
+        <ul id="filters-type" class="collapsed">
```

Comment:
I'm not sure the type filters need to be collapsible, I can't imagine there will be too many of them.